pkg/minikube/cluster/cluster.go

```diff
@@ -300,9 +303,53 @@ func createVirtualboxHost(config MachineConfig) drivers.Driver {
 	return d
 }
 
+func isISONotCachedInMinikubeDir(isoFilePath string) bool {
+	if _, err := os.Stat(isoFilePath); os.IsNotExist(err) {
+		return true
+	}
+	return false
+}
+
+func getISOFilePathFromConfig(config MachineConfig) string {
+	split := strings.Split(config.MinikubeISO, "/")
```

Comment:
Yes important to use `filepath` so this works cross-platform (think backslash path separators on Windows).